### PR TITLE
Create q.json + more for email + pizza.is-a.dev is mine

### PR DESCRIPTION
**Pull request by oliik2013**

### CODE REVIEW

**Comment by oliik2013** [comment]
pls megre

File: domains/pizza.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "oliik2013",
+           "email": "oliik2013@oliik2013.is-a.dev",
+           "discord": "coolname4262"
+        },
+    
+        "record": {
+            "CNAME": "oliik2013.github.io"
```
Comment (Stef-00012) [inline domains/pizza.json:9]: Can you show a preview of your website?

File: domains/q.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "oliik2013",
+           "email": "oliik2013@oliik2013.is-a.dev",
+           "discord": "coolname4262"
+        },
+    
+        "record": {
+            "CNAME": "oliik2013.github.io"
```
Comment (Stef-00012) [inline domains/q.json:9]: Can you show a preview of your website?

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/q.json (hunk 2 above)
Comment (oliik2013) [reply domains/q.json:9]: https://oliik2013.github.io/sigma.pl/

File: domains/pizza.json (hunk 1 above)
Comment (oliik2013) [reply domains/pizza.json:9]: https://oliik2013.github.io/strona-flash/

**Comment by oliik2013** [comment]
plss megre

**Comment by oliik2013** [comment]
i provided the previer

**Comment by oliik2013** [comment]
@Stef-00012

**Comment by wdhdev** [comment]
@oliik2013 I have reasons to believe you are under the age of 13, as indicated by the year you have put in your username "2013". Also, after I asked your age in Discord DMs, you blocked me, which I believe is more than enough proof to believe you are 11 years old, not 21 years old as you claimed in DMs. 

This PR will obviously not be accepted as you are violating Discord & GitHub's terms of service. Also, next time, please don't harass our unpaid, volunteer, maintainers to merge your PR, there are others ahead of the queue.

**Pull request closed without merging**